packages/jokul/src/components/datepicker/DatePicker.tsx

```diff
@@ -180,6 +184,17 @@ export const DatePicker = forwardRef<HTMLInputElement, DatePickerProps>(
             [onChange, setError, setDate, setShowCalendar, minDate, maxDate],
         );
 
+        const clickInput = useCallback(() => {
+            setShowCalendar((isOpen) => {
+                if (!isOpen) {
+                    window.requestAnimationFrame(() => {
+                        calendarRef.current?.focus();
```

Comment:
Sørger for at kalender får focus når den åpnes. Dette må vi gjøre selv når det åpnes fra et annet sted enn `Popover.Trigger`.